data-generation/src/main/scala/thesis/SnapshotDelta.scala

```diff
@@ -104,11 +122,14 @@ object SnapshotDeltaObject {
 
     // Create, update or omit (delete) edges based on values are present or not.
     val newSnapshotEdges = joinedEdges.flatMap(outerJoinedEdge => outerJoinedEdge._2 match {
+      case (None, None) => throw new IllegalStateException("The full outer join f*ucked up")
```

Comment:
lol 😂 love it

Reply:
klarte ikke å sensurere riktig engang haha